src/functions/getRegeneratedEmails/framework/database/query-builder.ts

```diff
@@ -0,0 +1,17 @@
+import * as mysql from 'mysql2';
+
+export const buildGetRegeneratedEmailQuery = (appRef: number): string => {
+  const template = `
+    SELECT application_reference as appRef,
+           JSON_ARRAYAGG(
+                JSON_OBJECT(
+                    'new_email', new_email,
+                    'regenerated_date', regenerated_date,
+                    'new_language', new_language
+                )
```

Comment:
Can we alias these so they are more inline with current conventions i.e `newEmail`, `regeneratedDate` and `newLanguage`